### PR TITLE
Spark scale issue fix.

### PR DESCRIPTION
**Pull request by rabarona**
Configuration variables for SPARK  command. 

The solution for scale issue combines the removal of large broadcast variables and spark configuration. With these new options, users can set the number of cores per executor making the processing faster, set driver memory, set driver max result memory and overhead memory for both driver and executors.

### CODE REVIEW

File: duxbay.conf [hunk 1]
```diff
@@ -32,6 +32,11 @@ LIPATH=${LUSER}/ingest
 
 SPK_EXEC='400'
 SPK_EXEC_MEM='2048m'
+SPK_DRIVER_MEM=''
```
Comment (NathanSegerlind) [inline duxbay.conf:35]: so, is the idea that these correspond to unspecified or default values?

or does the user _have_ to update these values in duxbay.conf during installation?

Comment (rabarona) [reply duxbay.conf:35]: They should update with new values. That's going to depend on cluster capacity and YARN configuration.
Documentation still in progress.

Comment (NathanSegerlind) [reply duxbay.conf:35]: ok

Comment (NathanSegerlind) [reply duxbay.conf:35]: LGTM

**Pull request merged**